sway_nl.md

```diff
@@ -81,5 +79,5 @@ Voer ```sway``` uit vanuit een TTY. Sommige Display Managers zouden kunnen werke
 
 SleepyMario biedt ondersteuning in het Nederlands op IRC en GitHub, maar op onregelmatige tijdstippen. Laat aub. een berichtje achter en hij zal u z.s.m te woord staan.  
 
-Stuur een email naar [SleepyMario](theonesleepymario@gmail.com) voor suggesties wat betreft deze vertaling. 
+Suggesties voor een betere vertaling graag via [GitHub](https://github.com/sleepymario).  
```

Comment:
GitHub pull requests against Sway, like this one. They shouldn't reach out to you directly, I will loop you into the review here on GitHub.